src/test/java/uk/gov/hmcts/cmc/claimstore/jackson/serialize/ClaimantResponseSerializationTest.java

```diff
@@ -1,33 +1,71 @@
 package uk.gov.hmcts.cmc.claimstore.jackson.serialize;
 
-import org.junit.Ignore;
 import org.junit.Test;
 import uk.gov.hmcts.cmc.ccd.config.CCDAdapterConfig;
 import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
-import uk.gov.hmcts.cmc.domain.models.Claim;
-import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
-import uk.gov.hmcts.cmc.domain.utils.ResourceReader;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseRejection;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimantResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponseAcceptation;
 
 import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasNoJsonPath;
 import static com.jayway.jsonpath.matchers.JsonPathMatchers.isJson;
-import static org.junit.Assert.assertEquals;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
 import static org.junit.Assert.assertThat;
 
-@Ignore
+
 public class ClaimantResponseSerializationTest {
 
     private JsonMapper processor = new JsonMapper(new CCDAdapterConfig().ccdObjectMapper());
+    private static final String JASON_PATH_PREFIX = "$.";
```

Comment:
JSON.... pls. JASON seems person name

Reply:
trying to be humane with the code !! ... (was a typo ;-)